flit-client/src/exportedTypes/index.ts

```diff
@@ -0,0 +1,7 @@
+import User from './User';
+import Message from './Message';
+import Conversation from './Conversation';
+
+export type FlitUser = User; 
+export type FlitMessage = Message;
+export type FlitConversation = Conversation;
```

Comment:
Similar to the constants, we might have an easier time maintaining this if we just keep these all in one file right now.